Build fails on Windows due to missing `usleep`

Trying to run `make` or `make player` fails with the following: 

```
$ make
g++ player.cpp -o player -lstdc++ -std=c++11
player.cpp: In function ‘int main()’:
player.cpp:154:4: error: ‘usleep’ was not declared in this scope; did you mean ‘sleep’?
  154 |    usleep(100000u/header.framerate);
      |    ^~~~~~
      |    sleep
make: *** [Makefile:26: player] Error 1
```

Tested with gcc on cygwin (`gcc (GCC) 9.2.0`) and MinGW (`gcc.exe (x86_64-posix-seh-rev0, Built by MinGW-W64 project) 8.1.0`).

A fix would be using `std::this_thread::sleep_for` from `<thread>` if on a Windows system.

--- a/player.cpp
+++ b/player.cpp
@@ -7,6 +7,11 @@
 #include <unistd.h>
 #include "data_header.h"
 
+#if defined(_WIN32) || defined(__CYGWIN__)
+#include <thread>
+#define usleep(t) std::this_thread::sleep_for(std::chrono::microseconds(t));
+#endif
+
 using namespace std::chrono;
 
 struct term_size_t {

--- a/player.cpp
+++ b/player.cpp
@@ -3,14 +3,10 @@
 #include <csignal>
 #include <cstring>
 #include <chrono>
+#include <thread>
 #include <assert.h>
 #include <unistd.h>
 #include "data_header.h"
-
-#if defined(_WIN32) || defined(__CYGWIN__)
-#include <thread>
-#define usleep(t) std::this_thread::sleep_for(std::chrono::microseconds(t));
-#endif
 
 using namespace std::chrono;
 
@@ -156,7 +152,7 @@ int main()
 			// but I don't actually want to sleep by an entire frame's duration
 			// I just want to sleep enough so the loop isn't constantly getting slammed
 			// while hopefully not actually stalling the framerate performance
-			usleep(100000u/header.framerate);
+			std::this_thread::sleep_for(microseconds(100000u/header.framerate));
 		}
 	}
 

